p2pserver/net/netserver/netserver.go

```diff
@@ -166,11 +166,11 @@ func (this *NetServer) handleFeed(event *dt.FeedEvent) {
 	case dt.Add:
 		node := event.Event.(*dt.Node)
 		address := node.IP + ":" + strconv.Itoa(int(node.TCPPort))
-		this.Connect(address, false)
+		go this.Connect(address, false)
```

Comment:
No need to use go routine. Because that connect has much lock, concurrent maybe impacted.